### PR TITLE
Run copy deduplicate DAG in shared-prod

### PR DESCRIPTION
**Pull request by relud**
fixes https://github.com/mozilla/bigquery-etl/issues/1614

### CODE REVIEW

File: dags/utils/gcp.py [hunk 1]
```diff
@@ -533,7 +533,7 @@ def bigquery_etl_copy_deduplicate(
     priority="INTERACTIVE",
     hourly=False,
     slices=None,
-    gcp_conn_id="google_cloud_derived_datasets",
+    gcp_conn_id="google_cloud_shared_prod",
```
Comment (relud) [inline dags/utils/gcp.py:536]: this impacts the copy_deduplicate jobs

File: dags/copy_deduplicate.py [hunk 2]
```diff
@@ -172,7 +172,7 @@
 
     # Daily and last seen views on top of every Glean application.
 
-    gcp_conn_id = "google_cloud_derived_datasets"
+    gcp_conn_id = "google_cloud_shared_prod"
```
Comment (relud) [inline dags/copy_deduplicate.py:175]: this impacts the downstream `baseline_clients_(daily|last_seen)` jobs

File: bin/run [hunk 3]
```diff
@@ -79,6 +79,7 @@ init_connections() {
     gcp_conn=(
       "google_cloud_airflow_dataproc"
       "google_cloud_derived_datasets"
+      "google_cloud_shared_prod"
```
Comment (jklukas) [inline bin/run:82]: Did you newly set up this connection?

I assume changing the conn_id changes what kube cluster this connects to. Do we have a `bq-load-gke-1` cluster in shared-prod? I had assumed we'd need to use the existing conn_id, but specify a different project_id parameter for the queries.

Comment (relud) [reply bin/run:82]: > I had assumed we'd need to use the existing conn_id, but specify a different project_id parameter for the queries.

you are totally right. and it's `--billing-project` that we are missing.

File: dags/utils/gcp.py [hunk 4]
```diff
@@ -579,6 +582,7 @@ def bigquery_etl_copy_deduplicate(
         image=docker_image,
         arguments=["script/copy_deduplicate"]
         + ["--project-id=" + target_project_id]
+        + (["--billing-projects"] + list(bp) if billing_projects else [])
```
Comment (jklukas) [inline dags/utils/gcp.py:585]:
```suggestion
        + (["--billing-projects"] + list(billing_projects) if billing_projects else [])
```

**Review by jklukas — APPROVED** [review]

**Pull request merged**